03-viz.Rmd

```diff
@@ -1,40 +1,528 @@
+visual noise to remove. But there are a few things we must do to improve clarity, such as adding informative axis labels and making the font a more readable size.
+In order to add axis labels we use the `xlab` and `ylab` functions. To change the font size we use the `theme` function with the `text` argument:
+
+```{r 03-data-co2-line-2}
+co2_line <- ggplot(co2_df, aes(x = date, y = concentration)) + 
+                   geom_line() +
+                   xlab('Year') +
+                   ylab('Atmospheric CO2 (ppm)') + 
+                   theme(text = element_text(size = 18))
+```
+
+```{r 03-data-co2-line-2-print, echo = FALSE, results = 'hide', message = FALSE, warning = FALSE}
+print(co2_line)
+```
+
+Finally, let's see if we can better understand the oscillation by changing the visualization a little bit. Note that it is 
+totally fine to use a small number of visualizations to answer different aspects of the question you are trying to answer. We will accomplish
+this by using *scales*, another important feature of `ggplot2` that allow us to easily transform the different variables and set limits.
+We scale the horizontal axis by using the `scale_x_continuous` function, and the vertical axis with the `scale_y_continuous` function.
+We can transform the axis by passing the `trans` argument, and set limits by passing the `limits` argument. In particular, here we 
+will use the `scale_x_continuous` function with the `limits` argument to zoom in on just five years of data (say, 1990-1995):
+
+```{r 03-data-co2-line-3}
+co2_line <- ggplot(co2_df, aes(x = date, y = concentration)) + 
+                   geom_line() +
+                   xlab('Year') +
+                   ylab('Atmospheric CO2 (ppm)') + 
+                   scale_x_continuous(limits = c(1990, 1995)) +
+                   theme(text = element_text(size = 18))
+```
+
+```{r 03-data-co2-line-3-print, echo = FALSE, results = 'hide', message = FALSE, warning = FALSE}
+print(co2_line)
+```
+
+Interesting! It seems that each year, the atmospheric CO2 increases until it reaches its peak somewhere around April, decreases until around late September, 
+and finally increases again until the end of the year. In Hawaii, there are two seasons: summer from May through October, and winter from November through April.
+Therefore, the oscillating pattern in CO2 matches up fairly closely with the two seasons.
+
+
+### The island landmass dataset
+This dataset contains a list of Earth's land masses as well as their area (in thousands of square miles). **Question:** Are the continents (North / South America, Africa, Europe, Asia, Australia, Antarctica) Earth's 7 largest landmasses? If so, what are the next few largest landmasses after those?
+```{r 03-data-islands}
+# islands data 
+# (with some light wrangling to make it a dataframe with informative column names)
+islands_df <- enframe(islands)
```

Comment:
Again, this is some uncommon, un-introduced data wrangling our audience hasn't seen. I suggest you do this in a script and then just load the ready to use data.

Reply:
cleaned data in script, used `read_csv` to load in textbook